examples/morphology/main.go

```diff
@@ -0,0 +1,35 @@
+// Port of http://members.shaw.ca/el.supremo/MagickWand/grayscale.htm to Go
+package main
+
+/*
+#cgo CFLAGS: -I/usr/local/include/ImageMagick-6
+#cgo LDFLAGS: -L/usr/sbin/lib
```

Comment:
I'll remove them - there were from something else - I couldn't get go to look in the appropriate directory for the ImageMagick libraries. Eventually I gave up and copied the libraries to where go was looking for them. Any advice on https://stackoverflow.com/questions/38296291/how-to-persuade-go-to-find-c-libraries would be great.